src/components/UserPic.tsx

```diff
@@ -22,17 +23,11 @@ export const UserPic: React.FC<UserPicProps> = ({ src, email, size = 32, ...prop
 
     const onLoadError: React.ReactEventHandler<HTMLImageElement> = useCallback(({ currentTarget }) => {
         currentTarget.onerror = null;
-        currentTarget.src = '/anonymous.png';
```

Comment:
Why here we do nothing on error, but [here](https://github.com/taskany-inc/bricks/pull/334/files#diff-fc4e5466b03ed2b606adb7ab361ba653733fa7f0580d9c97b73126580d0dbb30R93) we are using Circle as fallback?